botometerAnalyser/pipelines/usersAnalysis.js

```diff
@@ -0,0 +1,225 @@
+const request = require('request-promise');
+const config = require('config');
+const d = require('debug');
+
+const utils = require('../utils');
+const cache = require('../cache');
+const graph = require('../graph');
+const { botometerQueue } = require('../queues/botometer');
+
+const logError = d('BotometerAnalyser:queryText:error');
+const debug = d('BotometerAnalyser:queryText:debug');
+
+
+const RETWEET_ANALYSIS = 0;
```

Comment:
Why uses a marker in job data instead of Bull named job?

Reply:
Replace by strings

Reply:
done